pkg/storage/bucket/client_test.go

```diff
@@ -127,3 +131,108 @@ func TestClientMock_MockGet(t *testing.T) {
 
 	wg.Wait()
 }
+
+func TestClient_ConfigValidation(t *testing.T) {
+	testCases := []struct {
+		name          string
+		cfg           Config
+		expectedError string
+	}{
+		{
+			name: "valid storage_prefix",
+			cfg:  Config{Backend: Filesystem, StoragePrefix: "helloworld"},
+		},
+		{
+			name:          "storage_prefix non-alphanumeric characters",
+			cfg:           Config{Backend: Filesystem, StoragePrefix: "hello-world!"},
+			expectedError: "storage_prefix contains invalid characters, it may only contain digits and English alphabet letters",
```

Comment:
Nit: You may want to use `ErrInvalidCharactersInStoragePrefix` to avoid repeating the message.